workspaces/linux-host/Makefile

```diff
@@ -0,0 +1,66 @@
+# Makefile
+#
+# AUTHORS
+#
+# The Veracruz Development Team.
+#
+# COPYRIGHT AND LICENSING
+#
+# See the `LICENSING.markdown` file in the Veracruz root directory for
+# licensing and copyright information.
+
+.PHONY: all doc clean fmt
+
+PROXY_ATTESTATION_SERVER_DB  = $(abspath proxy-attestation-server.db)
+TEST_PARAMETERS = DATABASE_URL=$(PROXY_ATTESTATION_SERVER_DB) \
+	VERACRUZ_POLICY_DIR=$(abspath ../linux-runtime/test-programs) \
+	VERACRUZ_TRUST_DIR=$(abspath crates/test-collateral) \
```

Comment:
Is `VERACRUZ_TRUST_DIR` misnamed?

Reply:
`VERACRUZ_TRUST_DIR` is the location where the certificates and keys are. Is the objection about the name or that it points to `test-collateral`? For workspaces the top-level `test-collateral` will not contain `.wasm` files, test data and policies.

Reply:
No objection, just a question.